main.go

```diff
@@ -37,22 +37,29 @@ func main() {
 
 	var tags []string
 
-	switch repo {
-	case "hub.docker.com":
+	switch {
+	case strings.Contains(repo, "hub.docker.com"):
 		t, err := retrieveFromDockerHub(image)
 		if err != nil {
 			fmt.Fprintln(os.Stderr, err)
 			os.Exit(1)
 		}
 
 		tags = t
-	case "quay.io":
+	case strings.Contains(repo, "quay.io"):
```

Comment:
This is expected to match only `quay.io`, but actual matches also `*.quay.io` or `quay.io.*.

```suggestion
	case repo == "quay.io":
```

Reply:
fixed in 557f9da